src/components/UserInfo/RecentSubmission.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Table } from "antd";
+import "antd/dist/reset.css"; // Import Ant Design styles
+import { SubmissionResponseInfo } from "../../types";
+
+interface Props {
+  submissions: SubmissionResponseInfo[];
+}
+
+const RecentSubmissions: React.FC<Props> = ({ submissions }, index) => {
+  const columns = [
+    {
+      title: "Problem_ID",
+      dataIndex: "problemId",
+      key: "problemId",
+    },
+    {
+      title: "Problem_Title",
+      dataIndex: "problemTitle",
+      key: "problemTitle",
+    },
+    {
+      title: "Submission_Date",
+      dataIndex: "timeSubmitted",
+      key: "timeSubmitted",
+    },
+    {
+      title: "Status",
+      dataIndex: "status",
+      key: "status",
+    },
+  ];
+
+  return (
+    <Table dataSource={submissions} columns={columns} pagination={false} />
```

Comment:
Consider using `timestampFormatter` in `utils/DateFormatter.ts` to convert the raw timestamp to formatted date string when display time in table.

Reply:
Solved